tests/gem/packet_test_data.py

```diff
@@ -82,7 +82,7 @@
         "seconds": 997492,
         "serial_number": 603,
         "device_id": 11,
-        "temperatures": [256, -5, 0, 0, 0, 0, 0, 0],
```

Comment:
The GitHub UI makes this a little hard to see in the PR view; do we have any tests with a valid temp?

Reply:
-5 is a valid temperature :)

Reply:
COVID shot gave me a fever.  Clearly I shouldn't have been reviewing code...

Reply:
Can we get at least one positive value in these tests as well so we cover all of our bases?